### ISSUE TITLE
Bug S3: Logo partially hidden on landing pages

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#727]
### Steps to reproduce
- Go to https://loconomics.com/welcome/san-francisco
### Screen shots
![screen shot 2017-11-13 at 12 07 38 pm](https://user-images.githubusercontent.com/1202838/32746735-65465550-c86b-11e7-8821-65db6cb4e0cb.png)

@IagoSRL should be a quick fix (for you) so putting it in Top Priorities.

**Comment by IagoSRL** [comment]
Reproducing that.. just on Chrome at *live*, Firefox OK. With Chrome with *master* copy looks fine :-/

**Comment by IagoSRL** [comment]
I rebuilt and uploaded just in case but nothing.

Found: An optimization in the minification process for CSS discovered a version of [this non standard behavior at Chrome](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size#compatNote_1).
The original source line says:
```
background-size: 200px;
```
It's merged into the short version for the 'background' property in the minified version of the file, and understood by Chrome as if was originally
```
background-size: 200px 200px;
```
rather than then correct default value, that is:
```
background-size: 200px auto;
```

So, better to specific both dimensions in `background-size` rather than rely on default value.

@joshdanielson I uploaded to *live* already, so can be tested easily against minified files. It looks good on Windows, let me know and we can close.
